src/common/ChromeTargetManager.ts

```diff
@@ -101,7 +101,12 @@ export class ChromeTargetManager extends EventEmitter implements TargetManager {
     this.#connection.on('sessiondetached', this.#onSessionDetached);
     this.#setupAttachmentListeners(this.#connection);
 
-    this.#connection.send('Target.setDiscoverTargets', {discover: true});
+    // TODO: remove `as any` once the protocol definitions are updated with the
+    // next Chromium roll.
+    this.#connection.send('Target.setDiscoverTargets', {
+      discover: true,
+      filter: [{type: 'tab', exclude: true}, {}],
+    } as any);
```

Comment:
happy to type it out as an explicit type but I guess any is okay as it's only till the next roll